processor/configuration/src/main/java/com/redhat/service/smartevents/processor/ValidatorUtils.java

```diff
@@ -0,0 +1,8 @@
+package com.redhat.service.smartevents.processor;
+
+public class ValidatorUtils {
+
+    public static String missingFieldMessage(String key) {
+        return "The supplied " + key + " parameter is not valid";
+    }
```

Comment:
I think the method name `missing` is a little bit confusing in combination with actual text `not valid`